### ISSUE TITLE
Dates in Norwegian

### ISSUE THREAD

**Issue by olangberg** [NUKnightLab/TimelineJS#774]
There is something wrong with the date format in Norwegian. 

The 24th of March is now in this format: 24. Mars, 2015. In correct Norwegian spelling, there is no capital letter in the name of the month. It should be like this: 24. mars 2015

Is this something that can be fixed?

**Comment by JoeGermuska** [comment]
Yes! The file [is here](https://github.com/NUKnightLab/TimelineJS/blob/master/source/js/Core/Language/locale/no.js) -- if you're willing, GitHub offers an "edit" button on the page:

![image](https://cloud.githubusercontent.com/assets/46313/6805194/b36222be-d20f-11e4-97bb-7ed0ae6a4b7c.png)

If it's really just lowercasing the month names, of course we can do it, but if you'd like to help out, please feel free. GitHub makes it pretty easy for you to offer the improvements back to us.
